data/dataset_alt.py

```diff
@@ -77,7 +77,9 @@ def __getitem__(self, idx: int) -> Union[Data, HeteroData]:
             )
 
             # Filter out users we already sampled
-            users_to_check = difference(users_to_check, old_users_to_check)
+            users_to_check = difference_1d(
```

Comment:
renamed it to difference_1d, because it only works on 1d sequences (that's also the terminology numpy uses)